lib/utils/webpack.config.js

```diff
@@ -117,6 +117,7 @@ module.exports = (program, directory, stage, webpackPort = 1500, routes = []) =>
             },
             __PREFIX_LINKS__: program.prefixLinks,
           }),
+          new ExtractTextPlugin('styles.css'),
```

Comment:
Oh hmmm... when building the HTML pages you mean? That does make sense.

Probably what you need to do is just copy the modules css loader config from `develop` to `build-html` as right now in `build-html` *all* css loaders are turned off. https://github.com/jakubrohleder/gatsby/blob/d6e5388ea0b5581e4422afd02110808716d13e02/lib/utils/webpack.config.js#L357-L373